packages/design-system/src/components/web-components/ds-autocomplete/__snapshots__/ds-autocomplete.test.tsx.snap

```diff
@@ -20,22 +20,22 @@ exports[`Autocomplete can add a custom class name to the error message 1`] = `
       >
         <label
           class="ds-c-label"
-          for="autocomplete---test"
-          id="autocomplete---test__label"
+          for="autocomplete--145"
+          id="autocomplete--145__label"
```

Comment:
Looks like the dynamic ids came back though. Did you try using `root-id`?

Reply:
Thanks for the nudge! I’ve just updated the tests to use `root-id`, so they should now be using a static value for consistency. :)